doc/source/rllib/index.rst

```diff
@@ -65,129 +65,11 @@ Feature Overview
 ----------------
```

Comment:
btw, i know this is out of scope, but i think we should add more disclaimer around our 'conda' instructions (conda is widely known but there exists many other packaging frameworks out there, we should call that out)